### PR TITLE
Python2 to Python3 code modernization

### PR DESCRIPTION
**Pull request by novialriptide**
Continuation from https://github.com/pygame/pygame/pull/2837

### CODE REVIEW

**Review by ankith26 — COMMENTED** [review]
Looking good so far! 🎉 
Are you done with the PR, or do you want to add more py2/py3 cleanups in this one? IG some more bits could still do with some modernisation, but that could happen in future PRs too

**Comment by novialriptide** [comment]
> Looking good so far! 🎉 
> Are you done with the PR, or do you want to add more py2/py3 cleanups in this one? IG some more bits could still do with some modernisation, but that could happen in future PRs too

I think this PR is ready to go

File: docs/reST/tut/tom_games3.rst [hunk 1]
```diff
@@ -71,15 +71,15 @@ control over your resources. Take this example of an image loading function::
...
       """ Load image and return image object"""
-      fullname = os.path.join('data', name)
+      fullname = os.path.join("data", name)
       try:
           image = pygame.image.load(fullname)
           if image.get_alpha() is None:
               image = image.convert()
           else:
               image = image.convert_alpha()
       except pygame.error, message:
```
Comment (Starbuck5) [inline docs/reST/tut/tom_games3.rst:81]: This code is wrong as well. Pygame.image.load emits FileNotFoundError, not pygame.error. And what is this message thing? Python3 gives me a syntax error from that.

File: docs/reST/tut/tom_games6.rst [hunk 2]
```diff
@@ -319,7 +319,7 @@ like this::
           pygame.display.flip()
 
 
-  if __name__ == '__main__': main()
+  if __name__ == "__main__": main()
```
Comment (Starbuck5) [inline docs/reST/tut/tom_games6.rst:322]: How about moving the main() under the if statement, that seems more conventional.

**Comment by novialriptide** [comment]
@ankith26 and I had a discussion in Discord and we've decided that it'll be better to remove the `from pygame.locals import *` as well

**Review by illume — APPROVED** [review]
lgtm :) Thank you kindly.

**Review by Starbuck5 — APPROVED** [review]
Alright, I've read through it all, and tested locally.

Thanks for this!

> @ankith26 and I had a discussion in Discord and we've decided that it'll be better to remove the from pygame.locals import * as well

In the examples? That could be a follow up PR.

**Pull request merged**